packages/grafana-ui/src/components/ConfirmButton/ConfirmButton.story.tsx

```diff
@@ -36,51 +18,69 @@ export default {
     docs: {
       page: mdx,
     },
+    knobs: {
+      disable: true,
+    },
+  },
+  args: {
+    buttonText: 'Edit',
+    confirmText: 'Save',
+    size: 'md',
+    confirmVariant: 'primary',
+    disabled: false,
+    closeOnConfirm: true,
+  },
+  argTypes: {
+    confirmVariant: { control: { type: 'select', options: ['primary', 'secondary', 'destructive', 'link'] } },
+    size: { control: { type: 'select', options: ['sm', 'md', 'lg'] } },
+    className: NOOP_CONTROL,
   },
 };
 
-export const basic = () => {
-  const { size, buttonText, confirmText, confirmVariant, disabled, closeOnConfirm } = getKnobs();
+interface StoryProps extends Partial<Props> {
+  buttonText: string;
+}
+
+export const Basic: Story<StoryProps> = (args) => {
   return (
     <>
       <div className="gf-form-group">
         <div className="gf-form">
```

Comment:
I think these `.gf-form-group`, `.gf-from` divs and the react fragment `<>` can be removed from all the stories.